### ISSUE TITLE
Feature request: install via pip

### ISSUE THREAD

**Issue by ma-sadeghi** [Cantera/cantera#622]
I'm not sure if here's the right place to post a feature request, so sorry about that.

We're thinking of using Cantera to handle mixtures in our pore network simulation package, [OpenPNM](https://github.com/PMEAL/OpenPNM/). The only issue is that Cantera is not `pip`-installable, and we'd like our package to be installed solely using `pip` for convenience. I wonder if you could provide support for `pip`. I'd be happy to contribute myself if that's not on your priority list.

Thanks,
Amin

**Comment by bryanwweber** [comment]
Hi Amin,

The problem is that this is not trivial to solve. The main reason is because Cantera has a compiled C module that provides the functionality in the Python interface. Therefore, the base C library on the user system must match (or at least be compatible) with the C library on the build machine. Some of these issues have been resolved relatively recently with wheels, but there are a lot of pitfalls. All of which is to say, we're looking at it, but there isn't a timeline for this to be released.

I'd suggest you check out our Conda package instead. It seems you're familiar with Anaconda already from your installation instructions. Anaconda actually recommend that all packages be installed with conda if at all possible to avoid some of the issues I mentioned previously, and also because until very recently, the conda manager didn't know how to handle pip-installed packages. I would be happy to help you develop a conda package for your project, please send me an email at the address on my profile if you're interested.

**Comment by T81** [comment]
Is it possible to install Cantera in a virtual environment? I do not have Anaconda installed.

**Comment by bryanwweber** [comment]
Yes, this should be possible, but you will have to build Cantera from source. It should be sufficient to specify the `python_cmd=/path/to/your/venv/bin/python` option in `cantera.conf`. See here: https://cantera.org/compiling/installation-reqs.html#sec-installation-reqs to get started with compiling.
